packages/manager/modules/emailpro/src/emailpro.service.js

```diff
@@ -32,6 +35,11 @@ export default class EmailPro {
     this.$timeout = $timeout;
     this.$translate = $translate;
 
+    this.apiRoutes = get(
+      EMAILPRO_CONFIG.API_ROUTES,
+      Environment.getRegion(),
```

Comment:
In `Angular.js` you could use `CoreConfig.getRegion()` provided by `@ovh-ux/manager-core`

Reply:
I used `manager-config` since it is a smaller package. But it makes sense to use `manager-core` in angular context. I have done the change. Thanks :)